### PR TITLE
Cleanup of sparklines and spark hovers

### PR DESCRIPTION
**Pull request by ldecicco-USGS**
Heading out for a few hours, if someone wanted to review this, it's mostly working, but I currently managed to lose the hover text on the sparklines.

### CODE REVIEW

File: scripts/process/process.order_sites.R [hunk 1]
```diff
@@ -7,9 +7,22 @@ process.order_sites <- function(viz = as.viz("order-sites")){
...
+    arrange(desc(dec_long_va)) 
   
+  reordered_sites$station_nm <- sapply(reordered_sites$station_nm, siteCapper)
+    
   saveRDS(reordered_sites, viz[['location']])
-}
\ No newline at end of file
+}
+
+siteCapper <- function(x) {
```
Comment (jread-usgs) [inline scripts/process/process.order_sites.R]: was this moved from somewhere else? I thought it was in the viz already

File: scripts/process/process.sparks.R [hunk 2]
```diff
@@ -22,15 +20,16 @@ process.gage_sparks <- function(viz = as.viz('gage-sparks')){
...
                        site_no = site.nos, stringsAsFactors = FALSE) %>% 
     mutate(class = "sparkline", 
            id = sprintf("sparkline-%s", site_no), 
            style = "mask: url(#spark-opacity);",
            onmouseover=sprintf("setBold('nwis-%s');", site_no), 
            onmouseout=sprintf("setNormal('nwis-%s');hovertext(' ');", site_no),
-           onclick=sprintf("openNWIS('%s', evt);", site_no),
-           onmousemove=sprintf("hovertext('USGS %s',evt);", site_no))
+           onclick=sprintf("openNWIS('%s', evt);", site_no)) %>%
+    left_join(select(sites, site_no, station_nm), by="site_no") %>%
```
Comment (jread-usgs) [inline scripts/process/process.sparks.R]: A little worried that this join happens before some mouse events are declared but after others.

**Comment by jread-usgs** [comment]
I'd say don't worry about the mouse text for the time being since we are going to layer a bunch more things on the sparks

File: scripts/process/process.order_sites.R (hunk 1 above)
Comment (dblodgett-usgs) [reply scripts/process/process.order_sites.R]: Could get it by sourcing another file, yeah.

File: scripts/process/process.sparks.R (hunk 2 above)
Comment (dblodgett-usgs) [reply scripts/process/process.sparks.R]: Yeah, I bet this is where they got lost? This is how I did it over in the sites: https://github.com/USGS-VIZLAB/hurricane-maria/blob/master/scripts/process/process.classified_sites.R#L26 Not sure the same kind of thing will work over here?

Comment (jread-usgs) [reply scripts/process/process.sparks.R]: The spark opacity revel (slider) is no longer working too (the mask id)

**Pull request merged**